scripts/docker-integration-tests/aggregator/m3aggregator.yml

```diff
@@ -327,6 +327,7 @@ aggregator:
     - 10s:2d
   maxNumCachedSourceSets: 2
   discardNaNAggregatedValues: true
+  aggregationLastValueAdjustTimestamp: true
```

Comment:
The wording here left me a little confused. In my mind the current behavior is adjusting the timestamp and the new behavior is not, but I can see how the aggregator would look at it differently. perhaps something like `keep` or `respect` instead of `adjust` would be less ambiguous? or something like `Snap` but with reverse value as @arnikola mentions below?

Reply:
Fair, perhaps we can call it `aggregationLastValueKeepLastTimestamp: true`?